lib/copy-sync/copy-sync.js

```diff
@@ -66,7 +66,10 @@ function copyFile (srcStat, src, dest, opts) {
     fs.copyFileSync(src, dest)
     fs.chmodSync(dest, srcStat.mode)
     if (opts.preserveTimestamps) {
-      return utimesSync(dest, srcStat.atime, srcStat.mtime)
+      // Cannot rely on previously fetched srcStat because atime is updated by
+      // the read operation: https://nodejs.org/docs/latest-v8.x/api/fs.html#fs_stat_time_values
+      const updatedSrcStat = fs.statSync(src)
+      fs.utimesSync(dest, updatedSrcStat.atime, updatedSrcStat.mtime)
```

Comment:
I just realized I removed a `return` statement here but it doesn't change the behavior (it was actually unnecessary). Do you want me to re-add it?